src/vcpkg/build.cpp

```diff
@@ -1112,8 +1112,14 @@ namespace vcpkg::Build
 
         abi_tag_entries.emplace_back("ports.cmake", paths.get_ports_cmake_hash().to_string());
         abi_tag_entries.emplace_back("post_build_checks", "2");
-        std::vector<std::string> sorted_feature_list = action.feature_list;
-        Util::sort(sorted_feature_list);
+        InternalFeatureSet sorted_feature_list = action.feature_list;
+        // Check that no "default" feature is present. Default features must be resolved before attempting to calculate
```

Comment:
Note that unlike https://github.com/microsoft/vcpkg-tool/pull/336 this does *not* insert "core" and rejects "default".

Reply:
The current documentation string for InternalFeatureSet indicates that `default` is valid:
```cpp
/// In an internal feature set, "default" represents default features and missing "core" has no semantic
```
How do we know that there won't be pseudo-features here?

Reply:
@ras0219-msft We could introduce `InternalResolvedFeatureSet` or something like that if you wanted, but pseudo-features getting into an ABI calculation is a bug (since it would result in the same input having different ABIs depending on whether the pseudo-feature was resolved first).